### DISCUSSION TITLE
Gogs 0.13.1 has been released!

### DISCUSSION THREAD

**Discussion by unknwon** [gogs/gogs#7876] [Announcements]
>[!cAUTION]
>If you're using pre-compiled release archives with SQLite, please skip this patch release and use newer patch releases instead!

Hello everyone,

Hope you have pleasant experiences with Gogs so far, and we’re excited to announce that Gogs 0.13.1 is released!

You can find binary downloads on our [website](https://gogs.io/docs/installation/install_from_binary) and the [GitHub release](https://github.com/gogs/gogs/releases/tag/v0.13.1) page.

The CHANGELOG for this release is also available on the [GitHub release](https://github.com/gogs/gogs/releases/tag/v0.13.1) page.

Please enjoy and happy coding!

Please comment on this thread if you spot any packaging issues for the release. For general feature requests and bug reports, please use our [curated issue forms](https://github.com/gogs/gogs/issues/new/choose) :)

### Backwards compatibility

You may upgrade your installation directly to this version if you're running Gogs 0.12.0 or newer. **Versions prior to 0.12.0 need to run 0.12.0 once** to complete necessary migrations before trying to upgrade to this version.

**Comment by eleanordoesntcode** [comment]
hey, the releases tab does not provide any build for Intel Macs, does this mean the architecture is dropped?

**Reply by unknwon** [reply]
No, it just means there is no more physical Intel-based Macs available as part of our current release infrastructure. 😞 We're looking into dropping some CGO usage (i.e. SQLite driver) so we can cross-compile a more comprehensive list of release archives.

**Reply by eleanordoesntcode** [reply]
Ok! Do you plan on making a build soon or not? Please don't take this as me asking you to hurry, I'm just wondering since I saw your project as being updatable in [Homebrew](https://github.com/Homebrew/homebrew-cask/) and I wonder if I should hold off updating to have both architectures.

**Reply by unknwon** [reply]
Hmm, I wish it could be done fast but never hold your breath haha, I would have to do some basic sanity check on the SQLite file compatibility between the two drivers. 

BTW It isn't officially support/maintained to use Gogs via Homebrew but if that has been working for you then that's great 😁 I see it is directly downloading files from GitHub releases ... yeah, totally feel that's a pain.

**Reply by unknwon** [reply]
Just created https://github.com/gogs/gogs/issues/7882, you may follow updates there if you're interested.

**Reply by unknwon** [reply]
Actually, I tried `CGO_ENABLED=1 GOARCH=amd64 TAGS=cert task release` on my M1, which seems to work! Will upload for 0.13.2.

Altho I suspect that's only possible with macOS, still need to get pure-Go SQLite driver running so we can cross-compile to most of the platforms).

**Reply by unknwon** [reply]
Release archives for 0.13.2 is now available https://github.com/gogs/gogs/releases/tag/v0.13.2

**Reply by eleanordoesntcode** [reply]
Nice, thanks a lot!
